Gemfile.lock

```diff
@@ -1,62 +1,60 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.1.1)
+    better_errors (2.5.0)
       coderay (>= 1.0.0)
-      erubis (>= 2.6.6)
+      erubi (>= 1.0.0)
       rack (>= 0.9.0)
-    byebug (9.0.6)
-    coderay (1.1.1)
-    commander (4.4.3)
-      highline (~> 1.7.2)
-    database_cleaner (1.6.1)
+    byebug (10.0.2)
+    coderay (1.1.2)
+    commander (4.4.7)
+      highline (~> 2.0.0)
+    database_cleaner (1.7.0)
     diff-lcs (1.3)
-    erubis (2.7.0)
-    event_sourcery (0.13.0)
-    event_sourcery-postgres (0.3.0)
-      event_sourcery (>= 0.10.0)
+    erubi (1.7.1)
+    event_sourcery (0.22.0)
+    event_sourcery-postgres (0.8.0)
+      event_sourcery (>= 0.14.0)
       pg
-      sequel (~> 4.38)
-    highline (1.7.8)
-    method_source (0.8.2)
-    mustermann (1.0.0)
+      sequel (>= 4.38)
+    highline (2.0.0)
+    method_source (0.9.2)
+    mustermann (1.0.3)
     pg (0.20.0)
-    pry (0.10.4)
+    pry (0.12.2)
       coderay (~> 1.1.0)
-      method_source (~> 0.8.1)
-      slop (~> 3.4)
-    pry-byebug (3.4.2)
-      byebug (~> 9.0)
+      method_source (~> 0.9.0)
+    pry-byebug (3.6.0)
+      byebug (~> 10.0)
       pry (~> 0.10)
-    rack (2.0.3)
-    rack-protection (2.0.0)
+    rack (2.0.6)
+    rack-protection (2.0.4)
       rack
-    rack-test (0.6.3)
-      rack (>= 1.0)
-    rake (12.0.0)
-    rspec (3.6.0)
-      rspec-core (~> 3.6.0)
-      rspec-expectations (~> 3.6.0)
-      rspec-mocks (~> 3.6.0)
-    rspec-core (3.6.0)
-      rspec-support (~> 3.6.0)
-    rspec-expectations (3.6.0)
+    rack-test (1.1.0)
+      rack (>= 1.0, < 3)
+    rake (12.3.1)
+    rspec (3.8.0)
+      rspec-core (~> 3.8.0)
+      rspec-expectations (~> 3.8.0)
+      rspec-mocks (~> 3.8.0)
+    rspec-core (3.8.0)
+      rspec-support (~> 3.8.0)
+    rspec-expectations (3.8.2)
```

Comment:
### [rspec-expectations](https://github.com/rspec/rspec-expectations)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 3.6.0 → 3.8.2

[_~~change-log~~, [source-code](http://github.com/rspec/rspec-expectations)_]





<details>
<summary>Commits</summary>

A change of **75** commits. See the full changes on [the compare page](https://github.com/rspec/rspec-expectations/compare/v3.6.0...v3.8.2).

These are the first 10 commits:
- (fb7f461) [Add configuration for max object formatter length](https://github.com/rspec/rspec-expectations/commit/fb7f461bc837ff1a497841cea0e6956f26287253)
- (8516bdc) [Bump version to 3.7.0.pre](https://github.com/rspec/rspec-expectations/commit/8516bdc7a4ee42d5f3669753a6b31a7fed01c576)
- (955d603) [Fix `Should.md` headings.](https://github.com/rspec/rspec-expectations/commit/955d603421fa50e739662983f8dc09b88ccaf731)
- (8c1e0e4) [Updates for frozen-string-literal compatibility. (#997)](https://github.com/rspec/rspec-expectations/commit/8c1e0e419125fe2cb1ddc8932c163dbc70e70668)
- (5c4ca95) [Update changelog.](https://github.com/rspec/rspec-expectations/commit/5c4ca95b030788e2f62ee82335ed5836bd83c8e8)
- (b1c83b7) [Use clearer variable names in documentation (features). (#10…](https://github.com/rspec/rspec-expectations/commit/b1c83b771b7f65b6e7f9ba8a34eda2c760148d3d)
- (1c9d5d9) [Updated travis build scripts (from rspec-dev) (#1003)](https://github.com/rspec/rspec-expectations/commit/1c9d5d960115ff0de803819580956a2615778a7d)
- (da6ae06) [Add receiver info to ChangeDetails#value_representation](https://github.com/rspec/rspec-expectations/commit/da6ae067462829a77023aa67d61de80013294ded)
- (ff1991c) [Support Module, nil, and objects that don't respond to `#cla…](https://github.com/rspec/rspec-expectations/commit/ff1991c70e1e546ebf86c8e5b7316522a6921f20)
- (7e3d9bd) [Add surrounding backquote to result of `#inspect_message_wit…](https://github.com/rspec/rspec-expectations/commit/7e3d9bd6c0fbceaba2d58645e2b8828a5a53401a)


</details>